virt/virt-learn-more-about-openshift-virtualization.adoc

```diff
@@ -1,109 +1,66 @@
-| xref:../virt/virtual_machines/virtual_disks/virt-creating-data-volumes.adoc#virt-customizing-storage-profile_virt-creating-data-volumes[Customizing the storage profile]
-| xref:../virt/virt-using-the-cli-tools.adoc#virt-using-the-cli-tools[Using  the CLI tools]
-| xref:../virt/live_migration/virt-live-migration.adoc#virt-live-migration[Using live migration]
-
-|
-| xref:../virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc#virt-creating-and-using-boot-sources[Creating boot sources and attaching them to templates]
-| Viewing xref:../virt/logging_events_monitoring/virt-logs.adoc#virt-logs[logs] and xref:../virt/logging_events_monitoring/virt-events.adoc#virt-events[events]
-|
-
-|
-| link:https://www.openshift.com/blog/updating-a-boot-source-image[Updating boot source templates]
-| xref:../virt/logging_events_monitoring/virt-monitoring-vm-health.adoc#virt-monitoring-vm-health[Monitoring virtual machine health]
-|
-
-|
-|===
-
-[id="virt-learn-more-developer"]
-== Virtual machine administrator / developer
-
-[options="header",cols="4*"]
-|===
-|Learn |Use |Manage |Additional resources
-
-| xref:../virt/about-virt.adoc#about-virt[Learn about {VirtProductName}]
-| xref:../virt/install/virt-enabling-virtctl.adoc#virt-enabling-virtctl[Enabling the `virtctl` client]
-| Viewing xref:../virt/logging_events_monitoring/virt-logs.adoc#virt-logs[logs] and xref:../virt/logging_events_monitoring/virt-events.adoc#virt-events[events]
-| xref:../virt/logging_events_monitoring/virt-collecting-virt-data.adoc#virt-collecting-virt-data[Getting Support]
-
-|
-| xref:../virt/virtual_machines/virt-create-vms.adoc#virt-create-vms[Creating virtual machines]
-| xref:../virt/logging_events_monitoring/virt-monitoring-vm-health.adoc#virt-monitoring-vm-health[Monitoring virtual machine health]
-|
-
-|
-| xref:../virt/virtual_machines/virt-manage-vmis.adoc#virt-manage-vmis[Managing virtual machines instances]
-| xref:../virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc#virt-managing-vm-snapshots[Creating and managing virtual machine snapshots]
-|
-
-|
-| xref:../virt/virtual_machines/virt-controlling-vm-states.adoc#virt-controlling-vm-states[Controlling virtual machine states]
-|
-|
-
-|
-| xref:../virt/virtual_machines/virt-accessing-vm-consoles.adoc#virt-accessing-vm-consoles[Accessing the virtual machine consoles]
-|
-|
-
-|
-| xref:../virt/virtual_machines/virt-managing-configmaps-secrets-service-accounts.adoc#virt-managing-configmaps-secrets-service-accounts[Pass configuration data to virtual machines using secrets, configuration maps, and service accounts]
-|
-|
-
-|
-|===
+You can install and configure a basic {VirtProductName} environment in order to explore its features and functionality.
+
+[NOTE]
+====
+Cluster configuration procedures require `cluster-admin` privileges.
+====
+
+[id="virt-getting-started-before-you-begin"]
```

Comment:
Not part of this PR, but in the section "Preparing your cluster for OpenShift Virtualization":

The sentence: "If your cluster uses worker nodes with different CPUs, live migration failures can occur because different CPUs have different capacities." should read,
"If your cluster uses worker nodes with different CPUs, live migration failures can occur because different CPUs have different capabilities. "

Reply:
made correection. Thanks @aglitke